src/core/Akka/Configuration/ConfigurationFactory.cs

```diff
@@ -55,6 +55,7 @@ public static Config ParseString(string hocon)
             return ParseString(hocon, null);
         }
 
+#if !ANDROID
```

Comment:
Pretty much. I'll be picking this up again within the next few days, it's
in our sprint
On Thu, Jan 7, 2016 at 11:49 AM Aaron Stannard <notifications@github.com>
wrote:

> In src/core/Akka/Configuration/ConfigurationFactory.cs
> <https://github.com/akkadotnet/akka.net/pull/1620#discussion_r49117090>:
>
> > @@ -55,6 +55,7 @@ public static Config ParseString(string hocon)
> >              return ParseString(hocon, null);
> >          }
> >
> > +#if !ANDROID
>
> Is it only the configuration stuff that needed to be IFDEF-ed out in
> order for this to work?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akkadotnet/akka.net/pull/1620/files#r49117090>.
>